test/e2e/adminUI/pages/fieldTypes/url.js

```diff
@@ -0,0 +1,39 @@
+var utils = require('../../../utils');
+
+module.exports = function UrlType(config) {
+	var self = {
+		selector: '.field-type-url[for="' + config.fieldName + '"]',
+		elements: {
+			label: '.FormLabel',
+			value: 'input[name="' + config.fieldName + '"]',
+		},
+		commands: [{
```

Comment:
I think there may be some formatting issue in this command block? Shows a bit misaligned here :-)